prometheus/operator/base/servicemonitor.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: monitoring.coreos.com/v1
+kind: ServiceMonitor
+metadata:
+  name: odhservicemonitor
+  labels:
+    team: opendatahub
+  namespace: opendatahub
+spec:
+  selector: {}
+  endpoints:
+    - port: web
```

Comment:
is "web" a fairly standard prometheus port name?  It seems to me like a port named "web" would be more inclined to be a web app?

Reply:
@crobby thats what is in the example since a lot use 8080 and yes it is an http endpoint

Reply:
Ok, fair enough